src/NosCore.GameObject/Character.cs

```diff
@@ -443,15 +446,15 @@ public void ChangeClass(CharacterClassType classType)
             Class = classType;
             Hp = MaxHp;
             Mp = MaxMp;
-            SendPacket(GenerateTit());
-            SendPacket(GenerateStat());
-            MapInstance.SendPacket(GenerateEq());
-            MapInstance.SendPacket(this.GenerateEff(8));
+            await SendPacket(GenerateTit());
+            await SendPacket(GenerateStat());
+            await MapInstance.SendPacket(GenerateEq());
+            await MapInstance.SendPacket(this.GenerateEff(8));
             //TODO: Faction
-            SendPacket(this.GenerateCond());
-            SendPacket(GenerateLev());
-            SendPacket(this.GenerateCMode());
-            SendPacket(new MsgPacket
+            await SendPacket(this.GenerateCond());
+            await SendPacket(GenerateLev());
+            await SendPacket(this.GenerateCMode());
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Add '.ConfigureAwait(false)' to this call to allow execution to continue in any thread.](https://app.codacy.com/manual/NosCoreIO/NosCore/pullRequest?prid=5262615)